### DISCUSSION TITLE
Sherlock cannot be found after installation

### DISCUSSION THREAD

**Discussion by rimzzlabs** [sherlock-project/sherlock#2289] [Q&A]
![image](https://github.com/user-attachments/assets/1798a4cd-8bb3-4382-9cf5-dc4c720f4269)

Im using endavour OS, but couldn't start it after I install it, please help

**Comment by ppfeister** [comment]
> Note: '/home/rimzzlabs/.local/bin' is not on your PATH environment variable. **These apps will not be globally accessible until your PATH is updated.** Run `pipx ensurepath` to automatically add it, or manually modify your PATH in our shell's config file.

**Reply by rimzzlabs** [reply]
I've run the `pipx ensurepath` command but still doesn't work. So I need to manually set this up.

But another weird case is that I can run `sherlock265`, did I install wrong package?

**Reply by ppfeister** [reply]
> I've run the pipx ensurepath command but still doesn't work.

Hm. Normally that should source the changed config as well. Are you running bash or zsh? (you can double check with `echo $SHELL` if unsure -- looks like zsh though).

> But another weird case is that I can run sherlock265

First I've heard of this myself, but it seems to be related to an implementation of h.265: https://github.com/strukturag/libde265
No idea if you installed it by accident, if it's installed by default, or maybe even as a dependency of another package you have, but it's unrelated to this project and can be ignored.
